packages/containers/sandbox/mockBackend/kit.js

```diff
@@ -87,6 +88,7 @@ function suggestionBig() {
 		function onEnd() {
 			console.log('onEnd', body);
 			cache.photos = {
+				cacheable: true,
```

Comment:
fix the trigger return so we do not have loading again and again on the same field